lib/Linting.js

```diff
@@ -154,15 +156,27 @@ Linting.prototype._formatIssues = function(issues) {
 
   }, []);
 
-  // (2) if affected element is not visible, then report it on root level
+  // (2) if affected element is not visible, then report it on root or participant level
   reports = map(reports, function(report) {
     const element = self._elementRegistry.get(report.id);
 
     if (!element) {
       report.isChildIssue = true;
       report.actualElementId = report.id;
 
-      report.id = self._canvas.getRootElement().id;
+      // (2.1) Is a participant referring to the current issue? Then display on participant
+      const participants = self._elementRegistry.filter((ele) => { return is(ele, 'bpmn:Participant'); }),

```

Comment:
We could cache the participants / processes referenced by ID to speed things up (i.e. move them before the `#map`). Filtering across all diagram elements is a large operation we should not repeat unnecessarily.

Reply:
Godd point - done